mrtt-ui/package.json

```diff
@@ -47,6 +47,23 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.29.4",
+    "husky": "^4.3.8",
+    "lint-staged": "^12.4.0",
     "prettier": "^2.6.2"
+  },
+  "lint-staged": {
+    "src/**/*.{json,md}": [
+      "prettier --write"
+    ],
+    "src/**/*.{js,jsx,tsx,ts}": [
+      "prettier --write",
+      "eslint --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
```

Comment:
I don't have an answer but I'm wondering if this needs to be here. If it's run when staging occurs, does it need to happen at-pre-commit too? Do you know what best practice is?

Reply:
I've always used lint-staged with pre-commit checks. This lets each commit be as clean as possible. we may be able to do without the `npm run lint` pre push, although I kind of like it as a safeguard.